idunn/geocoder/models/geocodejson.py

```diff
@@ -151,6 +152,14 @@ class Feature(BaseModel):
     context: Optional[Context]
 
 
+class IntentionType(Enum):
+    BRAND = "brand"
+    CATEGORY = "category"
+    PLACE = "place"
+    POI = "poi"
+    STREET = "street"
```

Comment:
As these values are theoretically exposed by the API, I think that we need to be particularly careful about the naming and restrain unwanted dependency with the current tagger implementation. 
For example, 
* `"street"` would also include intention about exact addresses, so `"address"` could be clearer ?
* `"place"` sounds confusing to me: would `"locality"` be correct ? or is it a generic fallback case, and then `"any_place" ` could be more explicit ?